adafruit_adxl37x.py

```diff
@@ -79,17 +98,26 @@ class ADXL375(adafruit_adxl34x.ADXL345):
 
     """
 
-    def __init__(self, i2c, address=None):
+    def __init__(self, i2c: busio.I2C, address: Optional[int] = None):
         super().__init__(
             i2c, address if address is not None else _ADXL375_DEFAULT_ADDRESS
         )
 
     @property
-    def range(self):
+    def acceleration(self) -> Tuple[int, int, int]:
+        """The x, y, z acceleration values returned in a 3-tuple in :math:`m / s ^ 2`"""
+        x, y, z = unpack("<hhh", self._read_register(_REG_DATAX0, 6))
+        x = x * _ADXL347_MULTIPLIER * _STANDARD_GRAVITY
+        y = y * _ADXL347_MULTIPLIER * _STANDARD_GRAVITY
+        z = z * _ADXL347_MULTIPLIER * _STANDARD_GRAVITY
+        return x, y, z
+
+    @property
+    def range(self) -> int:
```

Comment:
I'm torn for what to do here.  As is, it _technically_ works, so the correct typing should be `-> None`.  Is the intention to allow for silently ignoring the lack of implementation, as opposed to raising `NotImplementedError`?

Reply:
@tekktrik You're the expert here! I'll defer to you on this one. It throws the `NotImplementedError` when attempting to use it.

Reply:
Hmmm.... if you want the behavior to be that it throws `NotImplementedError` on both getting and setting, then I'd say it's fine to leave the typing as is because there's never really going to be a case where someone is expecting to get `int` and actually gets anything at all, so the potentially future-safe implementation of returning `int` seems fine.  If you want the get functionality of `range` to not fail but also not give anything useful, I would probably type this as returning `None`.  Either implementation works so up to you!

Reply:
I think we can leave it as it is for now then, and if it comes up in the future as an issue, we'll look at updating it!

Reply:
So is the decision to raise `Not ImplementatedError` from this property getter as well?